Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -958,6 +966,10 @@ command scriptUncomment
       exit scriptUncomment
    end if
    
+   local tOldLock
+   put the lockScreen into tOldLock
```

Comment:
the logic for handling unlock screen is incorrect. lockscreen doesn't behave like lockmessages. As long as your lock/unlock calls are balanced then you are good to go. As the code is now, if the screen is locked already then you will end up adding unbalanced lock screens.